core/schema/cache.graphqls

```diff
@@ -2,11 +2,13 @@
 scalar CacheID
 
 extend type Query {
-  "Constructs a cache volume for a given cache key."
+  """
+  Constructs a cache volume for a given cache key.
+  Example:
+  cacheVolume("myapp-cache")
```

Comment:
We should avoid referencing field and parameter names in the documentation because other SDKs have different naming conventions. This won't look right in Go or Python (`CacheVolume` and `cache_volume` respectively).

Reply:
See https://github.com/dagger/dagger/pull/4367#issuecomment-1382619290